src/renderer/library/components/catalog/GridView.tsx

```diff
@@ -147,17 +148,24 @@ class CatalogGridView extends React.Component<IProps, IState> {
                     )
                 }
                 {
-                    this.state.versionInfo ?
-                    <div className={stylesGlobal.new_version}>
+                    this.state.versionInfo && this.props.newVersionURL && this.props.newVersion ?
```

Comment:
Why this is in GridView Component ?

Reply:
not sure, I just reused the existing code. this PR is a good opportunity to move things around if it makes more sense architecturally

Reply:
Maybe move this "toast" -looking component into the bottom footer info area that already contains the version number. In the Zed code editor, a text link appears in the bottom bar without yes/no buttons, I can just choose to ignore it, or click on the link and download/activate the new version. I like this UX pattern because it means we are strongly encouraging users to update, but we are not nagging them with a toast notification that obscures the UI underneath. @llemeurfr what do you think?

Reply:
<img width="1171" alt="Screenshot 2024-05-10 at 11 08 07" src="https://github.com/edrlab/thorium-reader/assets/625201/d84b4e4e-9355-40e9-bbc7-388d261528b3">